hack/validate_cluster.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+set -xE
+
+trap display_status EXIT
+
+#############
+# FUNCTIONS #
+#############
+EXEC_COMMAND="kubectl -n rook-ceph exec $(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[0].metadata.name}') -- ceph --connect-timeout 3"
+
+function wait_for_daemon () {
+  timeout=90
+  daemon_to_test=$1
+  while [ $timeout -ne 0 ]; do
+    if eval $daemon_to_test; then
+      return 0
+    fi
+    sleep 1
+    let timeout=timeout-1
+  done
+
+  return 1
+}
+
+function test_demo_mon {
+  # shellcheck disable=SC2046
+  return $(wait_for_daemon "$EXEC_COMMAND -s | grep -sq quorum")
+}
+
+function test_demo_mgr {
+  # shellcheck disable=SC2046
+  return $(wait_for_daemon "$EXEC_COMMAND -s | grep -sq 'mgr:'")
+}
+
+function test_demo_osd {
+  # shellcheck disable=SC2046
+  return $(wait_for_daemon "$EXEC_COMMAND -s | grep -sq '1 osds: 1 up.*, 1 in.*'")
+}
+
+function test_demo_rgw {
+  # shellcheck disable=SC2046
+  return $(wait_for_daemon "$EXEC_COMMAND -s | grep -sq 'rgw:'")
+}
+
+function test_demo_mds {
+  echo "Waiting for the MDS to be ready"
+  # NOTE(leseb): metadata server always takes up to 5 sec to run
```

Comment:
git blame gives us the author, right? 
```suggestion
  # NOTE: metadata server always takes up to 5 sec to run
```